common/v2/features/DeFiZap/components/ZapIndicators.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { SPACING, COLORS } from 'v2/theme';
+
+import bullishIndicator from 'assets/images/defizap/indicators/bullishIndicator.svg';
+import bearishIndicator from 'assets/images/defizap/indicators/bearishIndicator.svg';
+
+const SContainer = styled.div`
+  display: flex;
+`;
+
+const SImage = styled.img`
+  height: 16px;
+  width: 16px;
+`;
+const SText = styled.p`
+  padding-left: ${SPACING.XS};
+  color: ${COLORS.PURPLE};
+  margin-bottom: 0px;
+  font-size: 16px;
```

Comment:
Can we use `Typography` for text and `FONT_SIZE` constant for font-sizes?
There may be more cases than this one

Reply:
This should be bold as well it seems